# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/snap/fractional-filter-ref.yaml
# Commit: 2b42abbdb0df
# Full Hash: 2b42abbdb0df38f31dfa1178fe3b5f773f8e4812
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-20 21:58:21
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 9fa6f71111d0 -r 2b42abbdb0df gfx/wr/wrench/reftests/snap/fractional-filter-ref.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/snap/fractional-filter-ref.yaml	Sun Feb 20 18:56:55 2022 +0000
@@ -0,0 +1,13 @@
+root:
+  items:
+    - type: stacking-context
+      bounds: [0, 0, 1920, 1200]
+      transform: translate(-45, 7)
+      items:
+      - type: stacking-context
+        filters: [opacity(0.9)]
+        items:
+          - text: "This should be pixel aligned!"
+            origin: 416.543499 160.008325
+            size: 16
+